zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/descriptors/ComponentTemplateDescriptor.java

```diff
@@ -8,7 +8,7 @@
 package io.camunda.exporter.schema.descriptors;
 
 public interface ComponentTemplateDescriptor {
-  String getTemplateClasspathFileName();
+  String getTemplateJson();
```

Comment:
💭 Let's think together here about this a bit

Let's say Operate will implement a `ComponentTemplateDescriptor` (say `OperateComponentTemplateDescriptor`), you want them in this implementation to write formatted json representation of this component template?

I don't think that's advisable. 

For me, we either keep it as we did (representing it in json files next to index templates) or revert it to how operate did it to begin with (filling it from configs), but not halfway.